oak_functions/sdk/test_utils/src/lib.rs

```diff
@@ -257,11 +257,17 @@ pub fn assert_response_body(response: Response, expected: &str) {
     )
 }
 
-/// Create some valid Wasm bytecode only calling Abi functions `read_request` and `write_request`.
-pub fn create_echo_wasm_module_bytes() -> Vec<u8> {
+/// Create Wasm bytecode from manifest path relative from the `oak_functions` directory.
+fn create_wasm_module_bytes(relative_manifest_path: &str, release: bool) -> Vec<u8> {
```

Comment:
Can this now be called in tests that use `compile_rust_wasm`? For instance: https://github.com/project-oak/oak/blob/201977e66a923a6a74cb5217bff6422423042bc2/oak_functions/loader/src/tests.rs#L114-L119.

Reply:
I think we could! 

However, I suggest before we switch here, we try working with the suggested Cargo Environment variable for a bit to see whether any unexpected problems come up.